src/components/TicketTypeFilterButton.tsx

```diff
@@ -11,16 +11,17 @@ export const TicketTypeFilterButton = ({ isSelected, children, ...rest }: Ticket
     {...rest}
     startIcon={isSelected ? <CheckBoxIcon /> : <CheckBoxOutlineBlankIcon />}
     variant={isSelected ? 'contained' : 'outlined'}
+    color={isSelected ? 'tertiary' : 'white'}
     sx={{
       justifyContent: 'start',
-      color: 'common.black',
-      bgcolor: isSelected ? undefined : 'background.default',
-      borderColor: `${rest.color}.main`,
-
+      color: 'primary.main',
```

Comment:
Riktig at `color` settes både som prop og i `sx` her? skulle tro at den ene overstyrer den andre 🤔 
Ser det ligger en egen `color` for `.MuiButton-startIcon` også, men det syns jeg gir mening om den skal ha annen farge